app/src/main/java/utils/ScheduleEvent.java

```diff
@@ -1,17 +1,21 @@
 package utils;
 
+import android.graphics.Color;
+
 import java.time.LocalTime;
 import java.util.ArrayList;
+import java.util.Calendar;
 
 public class ScheduleEvent {
 
     Activity activityType;
     String name;
-    LocalTime startTime;
-    LocalTime endTime;
-    ArrayList<Day> days;
+    Calendar startTime;
+    Calendar endTime;
+    //This is an arraylist of integers to correspond to days using the Calendar.MONDAY etc. mapping.
```

Comment:
uses same standard to represent days as the day picker library so easy to go to/from that view.